### PR TITLE
Release `@fluentui/react-toolbar` as stable

### PR DESCRIPTION
**Pull request by ling1726**
see title

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/Avatar-mount_base.html">1308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/Avatar-mount.html">1332</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/Button-mount_base.html">939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/Button-mount.html">926</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProvider-mount_base.html">1585</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProvider-mount.html">1598</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProviderWithTheme-mount_base.html">632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProviderWithTheme-mount.html">630</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">583</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">625</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/InfoButton-mount_base.html">567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/InfoButton-mount.html">565</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/MakeStyles-mount_base.html">1908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/MakeStyles-mount.html">1909</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/Persona-mount_base.html">3015</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/Persona-mount.html">2957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/SpinButton-mount_base.html">2489</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26384/perf-test-react-components/SpinButton-mount.html">2536</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `59.888 kB`<br />`16.652 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `187.431 kB`<br />`52.596 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.379 kB`<br />`11.322 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.069 kB`<br />`2.053 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a2669d5c922e29311b92660bd4a4d3802a6dffd4'>a2669d5c922e29311b92660bd4a4d3802a6dffd4</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (miroslavstastny) [inline change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json:2]: patch release is enough to get rid of the `rc` in package.json?

File: packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export { VerticalButton } from './ToolbarVerticalButton.stories';
 export { FarGroup } from './ToolbarFarGroup.stories';
 
 export default {
-  title: 'Preview Components/Toolbar',
+  title: 'Toolbar',
```
Comment (miroslavstastny) [inline packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx:22]:
```suggestion
  title: 'Components/Toolbar',
```
image

File: change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json (hunk 1 above)
Comment (ling1726) [reply change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json:2]: yup

File: packages/react-components/react-toolbar/stories/Toolbar/ToolbarSubtle.stories.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { TextBold24Regular, TextItalic24Regular, TextUnderline24Regular } from '@fluentui/react-icons';
-import { Toolbar, ToolbarToggleButton } from '@fluentui/react-components/unstable';
-import type { ToolbarProps } from '@fluentui/react-components/unstable';
+import { Toolbar, ToolbarToggleButton } from '@fluentui/react-components';
+import type { ToolbarProps } from '@fluentui/react-components';
 
 export const Subtle = (props: Partial<ToolbarProps>) => (
   <Toolbar aria-label="Subtle" {...props}>
```
Comment (miroslavstastny) [inline packages/react-components/react-toolbar/stories/Toolbar/ToolbarSubtle.stories.tsx:7]: What is `subtle` in this story?

File: packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ export { VerticalButton } from './ToolbarVerticalButton.stories';
...
   component: Toolbar,
```
Comment (miroslavstastny) [inline packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx:23]: Any plan to add props tables for `Toolbar` children components?

File: change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json (hunk 1 above)
Comment (miroslavstastny) [reply change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json:2]: TIL

**Review by miroslavstastny — APPROVED** [review]

File: packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx (hunk 4 above)
Comment (ling1726) [reply packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx:23]: Is done in #26382

File: packages/react-components/react-toolbar/stories/Toolbar/ToolbarSubtle.stories.tsx (hunk 3 above)
Comment (ling1726) [reply packages/react-components/react-toolbar/stories/Toolbar/ToolbarSubtle.stories.tsx:7]: It's a bug in the docs, will be fixed in #26382

**Pull request merged**